### PR TITLE
Part13 task done

### PR DESCRIPTION
**Pull request by alexkulesh**

### CODE REVIEW

File: FirstSolution/Part13Task/Task.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        public Square(int length, string color)
+        {
+            this.length = length;
+            this.color = color;
+            area = length * length;
+        }
+        
+        public void PrintInformation()
+        {
+            RecalcuteArea();
```
Comment (a-miakhedka) [inline FirstSolution/Part13Task/Task.cs:57]: Move calling of this method into Main

Comment (Alex19777) [inline FirstSolution/Part13Task/Task.cs]: Move calling of this method into Main

Comment (alexkulesh) [reply FirstSolution/Part13Task/Task.cs]: Done

File: FirstSolution/Part13Task/Task.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        public Square(int length, string color)
+        {
+            this.length = length;
+            this.color = color;
+            area = length * length;
+        }
+        
+        public void PrintInformation()
+        {
+            Console.WriteLine("{0} square side length is {1} and area equals {2}", color, length, area);
```
Comment (Alex19777) [inline FirstSolution/Part13Task/Task.cs]: Use $""

**Review by Alex19777 — APPROVED** [review]

File: FirstSolution/Part13Task/Task.cs (hunk 2 above)
Comment (alexkulesh) [reply FirstSolution/Part13Task/Task.cs]: Done

**Pull request merged**